doc/distributed-run.md

```diff
@@ -12,9 +12,9 @@ and start scenarios on all known nodes (except master).
 amoc_dist:do(my_scenario, 1, 100, Opts).
 ```
 
-Start `my_scenario` spawning 100 amoc users with IDs from range (1,100) inclusive.
+Start `my_scenario` spawning 100 amoc users with IDs from the range [1,100] inclusive.
 In this case sessions are going to be distributed across all nodes except master.
-At this moment users are distributed in round-robin fashion - user1 goes to slave1,
+At this moment users are distributed in a round&mdash;robin fashion - user1 goes to slave1,
```

Comment:
Not there, again, the tricky rules are for example more or less schematized in https://www.punctuationmatters.com/hyphen-dash-n-dash-and-m-dash/

It should be:
> [...] users are distributed in a round-robin fashion — [...]

That is, "round-robin" uses a common hyphen: hyphens are the one to go for compound words for example. And to separate sentences like a comma would do, in a much more emphatic way, the "—", called em-dash.

I'm sorry to torture you with this. Professional handicaps, I once earned money on typography, I might miss some tricky language grammar but not a typographic detail :joy: :stuck_out_tongue: